infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2Infrastructure.java

```diff
@@ -47,10 +46,13 @@
 
 import com.google.common.collect.Maps;
 
+import lombok.Getter;
+
 
 public class AWSEC2Infrastructure extends AbstractAddonInfrastructure {
 
-    public static final String INSTANCE_ID_NODE_PROPERTY = "instanceId";
+    @Getter
+    public final String instanceIdNodeProperty = "instanceId";
```

Comment:
Public variables ? Getter ? You  should side either on using public variable or using getter on private one.

Reply:
You are right. I change it to private.